Info/DetectorConstructionMessenger.cpp

```diff
@@ -98,5 +104,7 @@ void DetectorConstructionMessenger::SetNewValue(G4UIcommand* command, G4String n
   } else if (command == fCreateGeometryType) {
     fDetector->CreateGeometryFileFlag(true);
     fDetector->SetGeometryFileType(newValue);
+  } else if (command == fPressureInChamber) {
+    fDetector->SetPressureInChamber(newValue);
```

Comment:
compiler crashes because I made an error, sorry for that. 
It should go like that
fDetector->SetPressureInChamber(fPressureInChamber->GetNewDoubleValue(newValue))